apps/medon-fe/src/components/Appointments/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { useTranslation } from 'react-i18next';
+import { useState } from 'react';
+
+import { useAppSelector } from 'redux/hooks';
+import { getUserSelector } from 'redux/features/userSlice/userSelectors';
+import PatientListCard from 'components/PatientListCard';
+import { patientList } from 'utils/mock/patientList';
+import {
+  TimeIcon,
+  Wrapper,
+  TextTime,
+  ShowMore,
+} from 'components/Appointments/styles';
+import { timeMessages } from 'components/Appointments/types';
+import { roles } from 'utils/constants';
+
+export default function AppointmentsList() {
+  const { t } = useTranslation();
+  const user = useAppSelector(getUserSelector);
+
+  const sortedPatientList = patientList
```

Comment:
I dont like that it in constant, each rerender we will have recalculation

Reply:
can avoid recalculating the sorted 'patient list' on every render by using a useMemo hook to memoize the sorted list. This will only recalculate the sorted list when the patientList or user values change.